static/app/components/core/text/index.mdx

```diff
@@ -0,0 +1,280 @@
+---
+title: Text
+description: Text components display text content with various typography options and styling controls.
+source: 'sentry/components/core/text'
+resources:
+  js: https://github.com/getsentry/sentry/blob/master/static/app/components/core/text/text.tsx
+  a11y:
+    WCAG 1.4.3: https://www.w3.org/TR/WCAG22/#contrast-minimum
+    WCAG 1.4.4: https://www.w3.org/TR/WCAG22/#resize-text
+    WCAG 1.4.12: https://www.w3.org/TR/WCAG22/#text-spacing
+    WAI-ARIA Text: https://www.w3.org/WAI/ARIA/apg/patterns/
```

Comment:
Updating these with a few more and removing the WAI-ARIA link as the patterns are only for complex widgets.

```suggestion
    WCAG 1.4.3: https://www.w3.org/TR/WCAG22/#contrast-minimum
    WCAG 1.4.4: https://www.w3.org/TR/WCAG22/#resize-text
    WCAG 1.4.6: https://www.w3.org/TR/WCAG22/#contrast-enhanced
    WCAG 1.4.8: https://www.w3.org/TR/WCAG22/#visual-presentation
    WCAG 1.4.12: https://www.w3.org/TR/WCAG22/#text-spacing
```